packages/ui/package.json

```diff
@@ -46,6 +48,18 @@
         "types": "./dist/components/ui/*.d.ts",
         "default": "./dist/components/ui/*.js"
       },
+      "./theme": {
```

Comment:
question: why is the theme needed to be exported, especially this way? We use `PenumbraUIProvider` in the `app/app.tsx` in the DEX.

suggestion: let's just export the `theme` object from `PenumbraUIProvider/index.tsx` file